src/behavior.ts

```diff
@@ -24,147 +24,257 @@
  *  THE SOFTWARE.
  */
 
-// d3
-// import Selection = d3.Selection;
-import { Selection, AsterPlotData } from "./dataInterfaces";
-// powerbi.extensibility.utils.interactivity
-import { interactivityBaseService, interactivitySelectionService } from "powerbi-visuals-utils-interactivityutils";
+import { Selection as d3Selection } from "d3-selection";
+import { PieArcDatum as d3PieArcDatum } from "d3-shape";
+import powerbi from "powerbi-visuals-api";
+import { legendInterfaces, dataLabelInterfaces } from "powerbi-visuals-utils-chartutils";
+import { ColorHelper } from "powerbi-visuals-utils-colorutils";
+import { AsterDataPoint } from "./dataInterfaces";
+import * as asterPlotUtils from "./utils";
+
+import ISelectionId = powerbi.visuals.ISelectionId;
+import ISelectionManager = powerbi.extensibility.ISelectionManager;
+import LegendDataPoint = legendInterfaces.LegendDataPoint;
+import LabelEnabledDataPoint = dataLabelInterfaces.LabelEnabledDataPoint;
 
+const EnterCode = "Enter";
```

Comment:
For such case, instead of creating multiple variables, I advise to create a single `const enum`. That will logically group these values and will make it easier to maintain in case we will want to handle more events. E.g.:
```
const enum EventCode {
    ENTER = "Enter",
    SPACE = "Space",
}
```

Reply:
Done.